src/components/CompareResults/beta/RevisionRowExpandable.tsx

```diff
@@ -73,7 +75,11 @@ function RevisionRowExpandable(props: RevisionRowExpandableProps) {
           <Divider />{' '}
         </div>
         {!shouldDisplayGraph && (
-          <div className={`${styles.bottomSpace}`}>{singleRun}</div>
+          <div className={`${styles.bottomSpace}`}>
+            {singleRun} <br />
+            <b>Base application</b>: {baseApplication} <br />
+            <b>New application</b>: {newApplication}{' '}
```

Comment:
Should we avoid displaying them if it's not present?
![image](https://github.com/mozilla/perfcompare/assets/454175/cb855258-6f4c-40ef-8767-78f441c35017)